diff --git a/node_modules/@apollo/client/react/data/QueryData.js b/node_modules/@apollo/client/react/data/QueryData.js
index bbc5a7a..b14ec04 100644
--- a/node_modules/@apollo/client/react/data/QueryData.js
+++ b/node_modules/@apollo/client/react/data/QueryData.js
@@ -67,6 +67,10 @@ var QueryData = (function (_super) {
             return result;
         };
         _this.obsRefetch = function (variables) {
+            if(!_this.currentObservable) {
+                _this.initializeObservableQuery()
+            }
+            
             return _this.currentObservable.refetch(variables);
         };
         _this.obsFetchMore = function (fetchMoreOptions) { return _this.currentObservable.fetchMore(fetchMoreOptions); };
