diff --git a/node_modules/@apollo/client/apollo-client.cjs.js b/node_modules/@apollo/client/apollo-client.cjs.js
index 9712e2c..4a92b54 100644
--- a/node_modules/@apollo/client/apollo-client.cjs.js
+++ b/node_modules/@apollo/client/apollo-client.cjs.js
@@ -5580,7 +5580,11 @@ var QueryData = (function (_super) {
             var _a;
             (_a = _this.currentObservable) === null || _a === void 0 ? void 0 : _a.stopPolling();
         };
-        _this.obsSubscribeToMore = function (options) { return _this.currentObservable.subscribeToMore(options); };
+        _this.obsSubscribeToMore = function (options) {
+            if(_this.currentObservable) {
+                return _this.currentObservable.subscribeToMore(options);
+            }
+        };
         _this.onNewData = onNewData;
         return _this;
     }
diff --git a/node_modules/@apollo/client/react/data/QueryData.js b/node_modules/@apollo/client/react/data/QueryData.js
index 4a80d15..2e90a3e 100644
--- a/node_modules/@apollo/client/react/data/QueryData.js
+++ b/node_modules/@apollo/client/react/data/QueryData.js
@@ -80,7 +80,11 @@ var QueryData = (function (_super) {
             var _a;
             (_a = _this.currentObservable) === null || _a === void 0 ? void 0 : _a.stopPolling();
         };
-        _this.obsSubscribeToMore = function (options) { return _this.currentObservable.subscribeToMore(options); };
+        _this.obsSubscribeToMore = function (options) {
+            if(_this.currentObservable) {
+                return _this.currentObservable.subscribeToMore(options);
+            }
+        };
         _this.onNewData = onNewData;
         return _this;
     }
