diff --git a/node_modules/@apollo/client/react/data/QueryData.js b/node_modules/@apollo/client/react/data/QueryData.js
index 6dc2cb3..df65d31 100644
--- a/node_modules/@apollo/client/react/data/QueryData.js
+++ b/node_modules/@apollo/client/react/data/QueryData.js
@@ -70,6 +70,10 @@ var QueryData = (function (_super) {
             return _this.previousData.result = result;
         };
         _this.obsRefetch = function (variables) {
+            if(!_this.currentObservable) {
+                console.log('Creating a new query')
+                _this.initializeObservableQuery()
+            }
             return _this.currentObservable.refetch(variables);
         };
         _this.obsFetchMore = function (fetchMoreOptions) { return _this.currentObservable.fetchMore(fetchMoreOptions); };
